internal/approval.go

```diff
@@ -327,3 +341,23 @@ func getTeamNameFromTeamHandle(teams []*github.Team, v string) (string, error) {
 	}
 	return "", fmt.Errorf("Team with name or slug %q not found", v)
 }
+
+func splitMembers(members []*github.User, commits []*github.RepositoryCommit) ([]string, []string) {
```

Comment:
I think wrapping returned values into a struct would make it clear what these sliced represent.

Reply:
Why did you lose strong typing, from `[]*github.User` to `[]string`?